### PR TITLE
[zh] Remove Master/Slave terminology from stateless application tutor…

### PR DESCRIPTION
**Pull request by ydcool**
…ial.

Ref: #26141

### CODE REVIEW

File: content/zh/docs/tutorials/stateless-application/guestbook.md [hunk 1]
```diff
@@ -435,13 +329,14 @@ If you deployed this application to Minikube or a local cluster, you need to fin
...
+      Forwarding from 127.0.0.1:8080 -> 80
+      Forwarding from [::1]:8080 -> 80
       ```
 
 <!--
-1. Copy the IP address, and load the page in your browser to view your guestbook.
+1. load the page [http://localhost:8080](http://localhost:8080) in your browser to view your guestbook.
 -->
-2. 复制 IP 地址，然后在浏览器中加载页面以查看留言板。
+1. 在浏览器中加载 [http://localhost:8080](http://localhost:8080) 页面以查看留言板。
```
Comment (SataQiu) [inline content/zh/docs/tutorials/stateless-application/guestbook.md:339]:
```suggestion
2. 在浏览器中加载 [http://localhost:8080](http://localhost:8080) 页面以查看留言板。
```

File: content/zh/docs/tutorials/stateless-application/guestbook.md [hunk 2]
```diff
@@ -572,10 +463,10 @@ Deleting the Deployments and Services also deletes any running Pods. Use labels
 5. 运行以下命令以删除所有 Pod，Deployments 和 Services。
```
Comment (SataQiu) [inline content/zh/docs/tutorials/stateless-application/guestbook.md:463]: 独立段落，应重新从 1 开始编号，建议参考英文页面排版样式 https://deploy-preview-26726--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/guestbook/#cleaning-up

**Review by SataQiu — APPROVED** [review]
Thanks for your contribution @ydcool !
LGTM, except for some nits.
/approve

File: content/zh/docs/tutorials/stateless-application/guestbook.md (hunk 1 above)
Comment (ydcool) [reply content/zh/docs/tutorials/stateless-application/guestbook.md:339]: 这里英文原文中列表序号都是1

**Pull request merged**